src/styles/WordTestList.scss

```
@@ -0,0 +1,70 @@
+@import './variables/colors.scss';
+@import './variables/textStyles.scss';
+
+.babo {
+    display: flex;
+    justify-content: center;
+    border-bottom: 1px solid;
+    padding-bottom: 16px;
+    margin-bottom: 12px;
+    width: calc(100vw - 40px);
+    margin-left: auto;
+    margin-right: auto;
+    color: $koreasy-color-gray-03;
+}
+
+.test-button {
+    @include body01-bold;
+    background-color: $koreasy-color-primary;
+    color: $koreasy-color-white;
+    border: none;
+    margin-right: 12px;
+    margin-left: 20px;
+    width: calc((100vw - 52px) / 2);
+    height: 40px;
+    border-radius: 12px;
+    font-size: medium;
+}
+
+.learn-button {
+    @include body01-bold;
+    border: none;
+    background-color: $koreasy-color-gray-06;
+    color: $koreasy-color-primary;
+    width: calc((100vw - 52px) / 2);
+    margin-right: 20px;
+    height: 40px;
+    border-radius: 12px;
+    font-size: medium;
+}
+
+.word-test-list-view-name {
+    @include title01-heavy;
+    color: $koreasy-color-primary;
+    margin-left: 20px;
+    // margin-bottom: 12px;
+}
+
+.word-test-main-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.word-test-list-title {
+    padding-left: 0px;
+    align-items: center;
+}
+
+.big-title {
+    @include title02-bold;
+    color: $koreasy-color-black;
+    margin-left: 20px;
+    padding-bottom: 0px;
+    // margin-top: 12px;
+}
+
+.small-title {
+    @include body01-regular;
+    color: $koreasy-color-gray-01;
+    margin-left: 20px;
+}
```

```scss
<div className="test-button">
    <div className="learn-button />
</div>
```
위와 같이 태그 내에 태그가 있는 구조의 경우, 클래스 계층적으로 작성해주세요.
```suggestion
.test-button {
    @include body01-bold;
    background-color: $koreasy-color-primary;
    color: $koreasy-color-white;
    border: none;
    margin-right: 12px;
    margin-left: 20px;
    width: calc((100vw - 52px) / 2);
    height: 40px;
    border-radius: 12px;
    font-size: medium;
    
            .learn-button {
            @include body01-bold;
            border: none;
            background-color: $koreasy-color-gray-06;
            color: $koreasy-color-primary;
            width: calc((100vw - 52px) / 2);
            margin-right: 20px;
            height: 40px;
            border-radius: 12px;
            font-size: medium;
        }
}
```

---

`word-test-list-view-name` 태그에 `margin-top: 32px`가 누락되었습니다. 지금은 PageTitle의 비활성화 버그로 인해 공백이 있는 것처럼 보이지만 실제로 develop와 Merge를 한 다음에 웹사이트를 다시보게 된다면, margin-top이 존재하지 않게 됩니다. 위 클래스에 margin-top 값을 줘야한다고 봅니다.

위에 Mule님이 길게 지적한 사항으로 인해 suggestion이 어렵습니다. 따라서 아래와 같이 클래스 부분에 덮어씌워주시면 될 것 같습니다.
```scss
    margin: 32px 20px 0 20px;
```

이 경우 상단에는 32px의 공백과 좌-우에는 20px의 공백을 주게됩니다.